src/libs/actions/Report.js

```diff
@@ -616,6 +617,29 @@ function createOptimisticChatReport(participantList) {
     };
 }
 
+/*
+ * Creates an optimistic IOU report with a randomly generated reportID
+ */
+function buildOptimisticIOUReport(total, chatReportID, currency, locale) {
```

Comment:
The other optimistic creation functions use "create" instead of "build". Let's change it for consistency.

Reply:
I could be wrong, but I thought we all decided to switch all of these function names to `buildXYZ`... @mountiny @luacmartins am I remembering incorrectly?

Reply:
Yes, we decided to switch to build since it doesn't actually create anything it just builds the object. We should change the other one to `buildXYZ` as well.

Reply:
What, I thought we decided on create since it creates the object 😄, I have even changed the previous buildOptimisticChatReport to create https://github.com/Expensify/App/blob/2d6c15c62a8d5513d8d0289ef086bc67fd2de0d4/src/libs/actions/Report.js#L620-L650 😄

Reply:
I'm going to refactor them in this PR once and for all. Any objections?

Reply:
24 seconds have passed and nobody commented, seems like we agree 😄

Reply:
Go for it, I am in the creationist camp still tho :rage1: